README.md

```diff
@@ -9,39 +9,98 @@ description: "A collection of samples showing how to use the Azure SDK for Go."
 
 # Azure SDK for Go Samples
 
-azure-sdk-for-go-samples is a collection of sample usages of the current version of the [Azure/azure-sdk-for-go][][here](https://github.com/Azure-Samples/azure-sdk-for-go-samples/tree/main/services). The current version of the [Azure/azure-sdk-for-go][] is referring the service packages under the `/services` directory.
+This repo contains a collection of sample usages of the new version of [Azure/azure-sdk-for-go][]. All the samples are placed under [`sdk`](https://github.com/Azure-Samples/azure-sdk-for-go-samples/tree/main/sdk) folder and the folder structures are corresponding to the service packages under the `/sdk` directory of the [Azure/azure-sdk-for-go][] repo.
 
-We have a collection of sample usages of the new version of the [Azure/azure-sdk-for-go][] [here](https://github.com/Azure-Samples/azure-sdk-for-go-samples/tree/main/sdk). The new version of the [Azure/azure-sdk-for-go][] is referring the service packages under the `/sdk` directory.
-
-For general SDK help start with the [main SDK README][].
+For general SDK help, please refer to the [SDK README][].
 
 ## To run tests
 
-1. set up authentication (see following)
-1. `go test -v ./services/network/` (or any package)
+### Prerequisites
+
+You will need Go 1.18 and latest version of resource management modules.
+
+You will need to authenticate to Azure either by using Azure CLI to sign in or setting environment variables.
+
+#### Using Azure CLI to Sign In
+
+You could easily use `az login` in command line to sign in to Azure via your default browser. Detail instructions can be found in [Sign in with Azure CLI](https://docs.microsoft.com/cli/azure/authenticate-azure-cli).
+
+#### Setting Environment Variables
```

Comment:
If I use Azure CLI login to authenticate, the system env setting are not required I think, we can add a statement at the begin: If your code is going to run in a none-user-interactive environment, you can also authenticate by setting environment variables as below.

Reply:
Added.